numpy/lib/tests/test_function_base.py

```diff
@@ -2538,7 +2538,7 @@ def test_0d(self):
         assert_(y == 0)
 
         x = 5
-        y = piecewise(x, [[True], [False]], [1, 0])
```

Comment:
This didn't make any sense, and should have failed. The conditions should be the same shape as the `x`